### PR TITLE
Bump policy SDK

### PR DESCRIPTION
**Pull request by ereslibre**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -39,3 +39,6 @@ wasmtime-provider = "0.0.5"
 
 [dev-dependencies]
 assert-json-diff = "2.0.1"
+
+[patch.crates-io]
+kubewarden-policy-sdk = { git = "https://github.com/ereslibre/policy-sdk-rust", branch = "non-wasm" }
```
Comment (flavio) [inline Cargo.toml:44]: I think this is something we will have to get rid of

**Review by flavio — CHANGES_REQUESTED** [review]
LGTM, aside from the minor detail I commented about

File: Cargo.toml (hunk 1 above)
Comment (ereslibre) [reply Cargo.toml:44]: Yes, when https://github.com/kubewarden/policy-sdk-rust/pull/20 is merged and tagged, I will remove this patch, update the PR and tag a new release.

**Comment by ereslibre** [comment]
Merging on green.

**Pull request merged**